// Test case 16: Error: Invalid return type
func foo() int { return "hello"; }
func main() {}
